.gn

```diff
@@ -30,10 +30,6 @@ default_args = {
   # for now. See http://clang.llvm.org/docs/ControlFlowIntegrity.html
   is_cfi = false
 
-  # Disabled due to CRT linking problems in Windows when using prebuilt V8.
-  # Works in other configurations.
-  is_component_build = false
```

Comment:
Removing this is cargo build to fail on OSX in debug mode. It seems the comment here is not correct